File: mesrapp-backend/mesrapp/src/main/java/com/example/mesrapp/model/AcceptedUser.java
package com.example.mesrapp.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "accepted_users")
public class AcceptedUser {
    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    private Integer id;
    @Column(nullable = false)
    private String spotiId;    
    @Column(nullable = false)
    private String acceptedSpotiIds;
    private String acceptedMusicId;
    private String acceptedArtistId;
}

File: mesrapp-frontend/mesrapp/app/src/main/java/com/example/mesrapp/models/AcceptedUser.kt
package com.example.mesrapp.models

import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity
data class AcceptedUser (
    @ColumnInfo(name = "id", index = true) val id: String,
    @ColumnInfo(name = "spotiId") val spotiId: String,
    @ColumnInfo(name = "acceptedSpotiIds") val acceptedSpotiIds: String = "",
    @ColumnInfo(name = "acceptedMusicId") val acceptedMusicId: String = "",
    @ColumnInfo(name = "acceptedArtistId") val acceptedArtistId: String = "",
) {
    constructor(spotiId: String) : this(spotiId, "", "", "", "")
}
